internal/validators/status/validator.go

```diff
@@ -147,17 +147,28 @@ func (sv *statusValidator) Validate(ctx context.Context) (validators.Status, err
 }
 
 func (sv *statusValidator) listGhaStatuses(ctx context.Context) ([]*ghaStatus, error) {
-	combined, _, err := sv.client.GetCombinedStatus(ctx, sv.owner, sv.repo, sv.ref, &github.ListOptions{})
-	if err != nil {
-		return nil, err
+	var combined []*github.RepoStatus
+	page := 1
+	for {
+		c, _, err := sv.client.GetCombinedStatus(ctx, sv.owner, sv.repo, sv.ref, &github.ListOptions{PerPage: 100, Page: page})
```

Comment:
Could you please add also pagination logic to `ListCheckRunsForRef` as well? 🙏 

https://github.com/upsidr/merge-gatekeeper/pull/48/files#diff-5c0d013fe9e031ae272ee0c3d741f7929c2a7aaf9fae9b396d820968c65fb0f7L175

Reply:
done 👍